templates/example.gotxt

```diff
@@ -2,7 +2,15 @@
 
 {{- template "doc" .Doc -}}
 
-{{- codeBlock "go" .Code -}}
+{{- codeBlock "go" .Code }}
```

Comment:
nit: closing brace should be changed back to `-}}`

Reply:
Updated